src/command_modules/azure-cli-resource/azure/cli/command_modules/resource/_params.py

```diff
@@ -75,3 +75,6 @@
 
 register_cli_argument('tag', 'tag_name', options_list=('--name', '-n'))
 register_cli_argument('tag', 'tag_value', options_list=('--value',))
+
+register_cli_argument('lock', 'name', options_list=('--name', '-n'))
+register_cli_argument('lock create', 'level', options_list=('--lock-type', '-t'))
```

Comment:
Should the CLI use the term "level" at all? The portal seems to only mention "type".